### ISSUE TITLE
Exception when detect_face does not find bounding boxes

### ISSUE THREAD

**Issue by andresespinosapc** [davidsandberg/facenet#1035]
When running `export_embeddings.py` on a set of my own images, I get an exception for some images. This is due to the first line of the following segment:

https://github.com/davidsandberg/facenet/blob/096ed770f163957c1e56efa7feeb194773920f6e/contributed/export_embeddings.py#L150-L157

The problem is that the shape of `bounding_boxes` is `(0, 5)` for these images. What should I do in this case? Should I just omit the cropping (AKA all the code in that segment)? I think this should be handled by default.

**Comment by BayuSasongko** [comment]
yes, i also found this error. can you give me the Solutions sir ?

**Comment by BayuSasongko** [comment]
help
